mkinitcpio.conf

```diff
@@ -60,6 +60,7 @@ HOOKS=(base udev autodetect modconf block filesystems keyboard fsck)
 #COMPRESSION="xz"
 #COMPRESSION="lzop"
 #COMPRESSION="lz4"
+#COMPRESSION="zstd"
```

Comment:
I'm still wary of adding this to the .conf file, even commented, because the users might get the impression that we support zstd, which we don't yet. I was thinking about reaching to the Arch kernel maintainers, @heftig, @anthraxx, and others, to see if we could have the zstd patch merged across the board, so this wouldn't be an issue.